### PR TITLE
Adding Format Json button

### PR DESCRIPTION
**Pull request by sebastianpx**
Adding an item in the dropdown menu to format / re-indent the JSON query.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,6 +34,7 @@
             <a class="btn btn-small dropdown-toggle" data-toggle="dropdown" href="#"><i class="icon-wrench"></i></a>
             <ul class="dropdown-menu">
                 <li><a id="copy_as_curl" tabindex="-1" href="#">Copy as cURL</a></li>
+                <li><a id="format_json" tabindex="-1" href="#">Format JSON</a></li>
```
Comment (bleskes) [inline index.html]: Can we reword this into Auto indent JSON  ( to conform to the ctrl+I short but that does the same)?

**Comment by bleskes** [comment]
Thanks very much for the pull request. I left one comment on the code. If can change that, I'll pull it right it.

Boaz

**Comment by bleskes** [comment]
Looking good! can you please squash this into one commit and I will pull it in.

**Comment by sebastianpx** [comment]
Ok I think it should be good now ? ( Sorry I am not very used to the github pull request flow yet). Anyways, thanks a lot for the tool, it is very useful !

**Pull request merged**